src/config/dynamic-theme-fixes.config

```diff
@@ -6109,6 +6109,16 @@ INVERT
 
 ================================
 
+coze.com
+
+CSS
+.semi-spin-children * {
```

Comment:
It is preferred to be more specific with the selectors (`*` match every element). But if it works it's fine.